src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -42,9 +43,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws ServletException {
     try {
       String url = request.getParameter(URL_PARAMETER);
-      // TODO: add numComments parameter. Comment count will be hard-coded until then.
+      String numCommentsInput = request.getParameter(NUM_COMMENTS_PARAMETER);
+      // If nothing is inputted, length of list is default, otherwise it's what was inputted.
+      long numComments =
+          (numCommentsInput == null || numCommentsInput.trim().equals(""))
```

Comment:
If you are using jquery's slider, .value has a default value of 0:
https://api.jqueryui.com/slider/#option-value

So your condition should be updated, as we don't need to worry that it is null or "".

Also, .value returns a number, can we simply use do: 
```long numComments = request.getParameter(NUM_COMMENTS_PARAMETER)```?

Reply:
The former makes sense, however I believe the latter is not viable. If I understand correctly request.getParameter() returns a string regardless, since it's effectively deriving it from the URL.